NEWPART.md

```diff
@@ -0,0 +1,203 @@
+
+# Quick guide to adding a new part to an existing family
+
+This documents what I did to add support for xc7a100t parts, 
+in hope that it is useful to someone adding another new part.
+
+_If you are **just** adding a new package for a device that is already handled, you can skip Steps 2..4!_
+
+
+### Step 0 ###
+Fork a copy of https://github.com/SymbiFlow/prjxray on GitHub (go to the page, click "Fork" button, select your own workspace).
+
+Clone your fork (replace "tcal-x" with your GitHub ID):
+
+```
+git clone git@github.com:tcal-x/prjxray.git
```

Comment:
You may just point to the SymbiFlow/prjxray. There will be no need for instructing a user to replace your github id.

Reply:
I've replaced "tcal-x" with "<yourGithubId>", since I don't want them to start hacking directly on their clone of SymbiFlow's repo.   Yes, if they know enough git they will know how to update things so they can commit to their own fork later, but it seems simpler if they start with their own fork...